src/app/app-routing.module.ts

```diff
@@ -101,6 +109,11 @@ const routes: Routes = [
   { path: 'complaintupdate/:id', component: UpdateComplaintComponent},
   { path: 'compaintdetails/:id', component: ComplaintDetailsComponent},
 
+ { path: 'categorys', component: CategoryListComponent },
+  { path: 'categorys/add', component: CreateCategoryComponent },
```

Comment:
I dont understand Why categories are comming now. Those should be in expenses related PR right

Reply:
I forgot to add this file sir